crawl-ref/source/stairs.cc

```diff
@@ -769,6 +786,9 @@ void floor_transition(dungeon_feature_type how,
                 mpr(branches[branch].entry_message);
             else if (branch != BRANCH_ABYSS) // too many messages...
                 mprf("Welcome to %s!", branches[branch].longname);
+
+            if (how == DNGN_ENTER_GAUNTLET)
+                _gauntlet_effect();
         }
```

Comment:
I'd prefer this to be `if (branch == BRANCH_GAUNTLET)` I think.

There are two ways for the player to enter a gauntlet: through the portal, and from leaving the abyss. I think this message should be printed in both cases.

Reply:
I'll implement it, thanks.